src/styles/index.scss

```diff
@@ -1,4 +1,4 @@
-@import '~@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/css_variables.css';
+// @import '~@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/css_variables.css';
```

Comment:
on 99% this should be removed (leftovers after migration from fundamental to UI5)